arrow/util/protobuf_reflect.go

```diff
@@ -18,6 +18,8 @@ package util
 
 import (
 	"fmt"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/types/known/structpb"
```

Comment:
can you shift these includes to be grouped with the others of the same prefix below?